aries_cloudagent/revocation/models/issuer_rev_reg_record.py

```diff
@@ -307,6 +340,80 @@ async def send_entry(
 
         return rev_entry_res
 
+    async def fix_ledger_entry(
+        self,
+        profile: Profile,
+        apply_ledger_update: bool,
+        genesis_transactions: str,
+    ) -> (dict, dict, dict):
```

Comment:
I see a warning on my end here: Tuple expression not allowed in type annotation. Use Tuple[T1, ..., Tn] to indicate a tuple type